packages/calcite-components/src/components/panel/panel.scss

```diff
@@ -2,11 +2,30 @@
  * CSS Custom Properties
  *
  * These properties can be overridden using the component's tag as selector.
- *
- * @prop --calcite-panel-content-space: Specifies the padding of the component's content.
- * @prop --calcite-panel-footer-padding: Specifies the padding of the component's footer.
- * @prop --calcite-panel-header-border-block-end: Specifies the component header's block end border.
+
+ * @prop --calcite-panel-corner-radius: Specifies the corner radius of the component.
+ * @prop --calcite-panel-heading-text-color: Specifies the text color of the component's `heading`.
+ * @prop --calcite-panel-description-text-color: Specifies the text color of the component's `description`.
+ * @prop --calcite-panel-border-color: Specifies the border color of the component.
+
  * @prop --calcite-panel-background-color: Specifies the background color of the component.
+ * @prop --calcite-panel-header-background-color: Specifies the background color of the component's footer.
+ * @prop --calcite-panel-footer-background-color: Specifies the background color of the component's footer.
+
+ * @prop --calcite-panel-space: Specifies the padding of the component's `"unnamed (default)"` slot.
+ * @prop --calcite-panel-header-content-space: Specifies the padding of the `"header-content"` slot.
+ * @prop --calcite-panel-footer-space: Specifies the padding of the component's footer.
+
+ * @prop --calcite-panel-content-space: [Deprecated] Specifies the padding of the component's content. Use `--calcite-panel-space` instead.
+ * @prop --calcite-panel-footer-padding: [Deprecated] Specifies the padding of the component's footer. Use `--calcite-panel-footer-space` instead.
+ * @prop --calcite-panel-header-border-block-end: [Deprecated] Specifies the component header's block end border. Use `--calcite-panel-border-color` instead.
```

Comment:
this appears to be overriding `--calcite-panel-border-color`

Reply:
Swapped the order of these.